test/jobs/single_submission_job_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
You can delete these files for now. We will ask you to write tests in a subsequent pull request.